src/main/java/cm/ptks/craftflowers/flower/FlowerRegistry.java

```diff
@@ -1,170 +1,174 @@
+		registerFlower(new Flower(Material.BRAIN_CORAL_FAN, LanguageFile.FLOWER_BRAIN_CORAL_FAN));
+		registerFlower(new Flower(Material.BUBBLE_CORAL_FAN, LanguageFile.FLOWER_BUBBLE_CORAL_FAN));
+		registerFlower(new Flower(Material.FIRE_CORAL_FAN, LanguageFile.FLOWER_FIRE_CORAL_FAN));
+		registerFlower(new Flower(Material.HORN_CORAL_FAN, LanguageFile.FLOWER_HORN_CORAL_FAN));
+		registerFlower(new Flower(Material.TUBE_CORAL_FAN, LanguageFile.FLOWER_TUBE_CORAL_FAN));
+
+		registerFlower(new FlowerGroup(Material.WHEAT, LanguageFile.FLOWER_WHEAT, Arrays.asList(
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_0, 0),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_1, 1),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_2, 2),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_3, 3),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_4, 4),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_5, 5),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_6, 6),
+				new AgingFlower(Material.WHEAT, LanguageFile.FLOWER_WHEAT_AGE_7, 7)
+		)));
+
+		registerFlower(new FlowerGroup(Material.POTATO, LanguageFile.FLOWER_POTATO, Arrays.asList(
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_0, Material.POTATOES, 0),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_1, Material.POTATOES, 1),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_2, Material.POTATOES, 2),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_3, Material.POTATOES, 3),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_4, Material.POTATOES, 4),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_5, Material.POTATOES, 5),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_6, Material.POTATOES, 6),
+				new AgingFlower(Material.POTATO, LanguageFile.FLOWER_POTATO_AGE_7, Material.POTATOES, 7)
+		)));
+
+		registerFlower(new FlowerGroup(Material.CARROT, LanguageFile.FLOWER_CARROT, Arrays.asList(
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_0, Material.CARROTS, 0),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_1, Material.CARROTS, 1),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_2, Material.CARROTS, 2),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_3, Material.CARROTS, 3),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_4, Material.CARROTS, 4),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_5, Material.CARROTS, 5),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_6, Material.CARROTS, 6),
+				new AgingFlower(Material.CARROT, LanguageFile.FLOWER_CARROT_AGE_7, Material.CARROTS, 7)
+		)));
+		registerFlower(new Flower(Material.BARRIER, LanguageFile.FLOWER_BARRIER, Material.AIR));
```

Comment:
FLOWER_AIR is better here since it actually places air and the barrier is only the icon